SplunkRumWorkspace/SplunkRum/SplunkRum/Session.swift

```diff
@@ -23,6 +23,7 @@ private var rumSessionId = generateNewSessionId()
 private var sessionIdExpiration = Date().addingTimeInterval(TimeInterval(MAX_SESSION_AGE_SECONDS))
 private let sessionIdLock = NSLock()
 private var sessionIdCallbacks: [(() -> Void)] = []
+private var oldRumSessionId = ""
```

Comment:
This can *also* be a local variable inside `getRumSessionId`.

Reply:
keep `oldRumSessionId` in side `getRumSessionId`.